README.md

```diff
@@ -100,16 +108,47 @@ A "backport" release, [1.13.2](https://github.com/TASVideos/BizHawk/releases/tag
 
-The systems that currently work are: GB + GBC (GBHawk), NES (NesHawk), SMS, Atari 7800, and some classic home computers. See [#1430](https://github.com/TASVideos/BizHawk/issues/1430) for progress.
+Most features and cores work, notable omissions being Lua support, Mupen64Plus (N64), and Octoshock (PSX). See [#1430](https://github.com/TASVideos/BizHawk/issues/1430) for details.
 
 [to top](#bizhawk)
 
+#### macOS (legacy BizHawk)
+
+EmuHawk depends on certain libraries for graphics, and these don't work on macOS. Users on macOS have three options:
+* Use another machine with Windows or Linux, or install either in a VM (WINE is not a VM).
+* Use an older 1.x release which was ported to macOS by @Sappharad (with replacements for the missing libraries). Links and more details are in [this TASVideos forum thread](http://tasvideos.org/forum/viewtopic.php?t=12659) (jump to last page for latest binaries).
+* For the technically-minded, download the [source](https://github.com/Sappharad/BizHawk/tree/MacUnixMonoCompat) of an older 2.x release. @Sappharad put a lot of work into it but ultimately decided to stop.
+
+[to top](#bizhawk)
+
+### Development builds
+
+Development builds are made automatically whenever someone contributes. Because of this, we recommend using a release for work that requires stability (such as TASing), and only switching to a dev build if there's a specific change or addition you need.
+
+[![recent dev builds | AppVeyor](https://img.shields.io/badge/recent_dev_builds-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/history)
+[![recent dev builds | GitLab CI](https://img.shields.io/badge/recent_dev_builds-GitLab_CI-orange.svg?logo=gitlab&logoColor=333333&style=popout)](https://gitlab.com/TASVideos/BizHawk/pipelines)
+[![latest build of master | GitLab CI](https://img.shields.io/badge/latest_build_of_master-GitLab_CI-orange.svg?logo=gitlab&logoColor=333333&style=popout)](https://gitlab.com/TASVideos/BizHawk/pipelines/master/latest)
+
+Click one of the buttons above to download a dev build (they're also at the top of this readme). AppVeyor uses Windows and GitLab CI uses Linux, but they work all the same.
+* On the AppVeyor page for a Build, click "Artifacts", then `BizHawk_Developer-<datetime>-#<long hexadecimal>.zip`.
+* On the GitLab CI page for a Pipeline, click "Jobs", then the download button at the end of the entry with the name `build_master`/`build_not_master`. (On the Pipelines list page, there's also a download button on each Pipeline.)
```

Comment:
Including test_output and PDBs into the main build makes it inconsistent with appveyor output.

Reply:
Need to discuss 1st point in IRC. 2nd point is a good idea (it currently uses the branch/tag name, or commit if run manually and not on a branch).